src/components/ReportActionItem/MoneyRequestPreview.js

```diff
@@ -151,8 +152,9 @@ function MoneyRequestPreview(props) {
     // Pay button should only be visible to the manager of the report.
     const isCurrentUserManager = managerID === sessionAccountID;
 
-    const {amount: requestAmount, currency: requestCurrency, comment: requestComment, merchant: requestMerchant} = ReportUtils.getTransactionDetails(props.transaction);
-    const description = requestComment;
+    const {amount: requestAmount, currency: requestCurrency, comment: requestComment, merchant} = ReportUtils.getTransactionDetails(props.transaction);
```

Comment:
Why are we changing `, merchant: requestMerchant}` to just `, merchant}`??

Reply:
`requestMerchant` is defined below Line 157

Reply:
Then do we even need the `,merchant}` in here anymore? The value for `merchant` that gets returned from `getTransactionDetails` isn't being used anywhere anymore...is it?